packages/components/src/components/tag/_tag.scss

```diff
@@ -27,19 +27,19 @@
     );
 
     display: inline-flex;
-    align-items: center;
-    justify-content: center;
     // ensures tag stays pill shaped;
     min-width: rem(32px);
-    // restricts size of contained elements
     max-width: 100%;
     min-height: rem(24px);
-    margin: $carbon--spacing-02;
+    align-items: center;
+    justify-content: center;
     padding: $carbon--spacing-02 $carbon--spacing-03;
-    vertical-align: middle;
-    word-break: break-word;
+    // restricts size of contained elements
```

Comment:
should this comment have moved too?

Reply:
I'm unsure if it was a comment for `max-width` or something else. Just kind of confused me. Disregard if it's correct 😅

Reply:
thanks, updated